### DISCUSSION TITLE
Issue title categorization with `[]` and `{}`

### DISCUSSION THREAD

**Discussion by zackkrida** [WordPress/openverse#95] [Q&A] [answered]
We currently use bracketed text to give extra info about our issues. This includes things like `[Feature]` `{Audio}` and so on. The usage of these labels seems potentially cluttering and that it could contradict with our `aspect` and `goal` labels. 

We should likely remove these or standardize their usage.

CC: @WordPress/openverse-developers

**Comment by AetherUnbound** [comment]
Agreed, I think the only thing that might need to be added (for the catalog repo at least) is a `goal: infrastructure improvement`, unless we'd like to fold that into `goal: internal improvement`. We have a separate issue type for it, so it would seem to warrant a separate goal too

**Reply by dhruvkb** [reply]
Speaking from a place of little experience, it seems that infrastructure improvements a subset of internal improvements. Would it be possible to do the opposite, i.e. merge the issue types instead of the labels?

**Reply by AetherUnbound** [reply]
That's a fair point! I guess I don't have any strong opinions either way on this one :slightly_smiling_face: But I agree some parity should be achieved one way or another.

**Comment by sarayourfriend** [comment, marked as answer]
I'd be down to ditch them in favor of the labels we already have. The Gutenberg repository (a much larger one than any of ours currently) doesn't use them either (except for flaky tests it looks like? But I'm not sure why there's a difference there): https://github.com/WordPress/gutenberg/issues

**Comment by dhruvkb** [comment]
PR: https://github.com/WordPress/openverse/pull/96
